docs/src/modules/components/AppFrame.js

```diff
@@ -154,7 +154,7 @@ const StyledAppNavDrawer = styled(AppNavDrawer)(({ disablePermanent, theme }) =>
 });
 
 export default function AppFrame(props) {
-  const { children, disableDrawer = false, className } = props;
+  const { children, disableDrawer = false, className, BannerComponent } = props;
```

Comment:
```suggestion
  const { children, disableDrawer = false, className, BannerComponent = AppFrameBanner } = props;
```
Maybe this would be more explicit than `defaultProps`?